### PR TITLE
[UNR-890] - Tidy Up Content

### PR DESCRIPTION
**Pull request by jnicholas-io**
Changed name of StarterProjectCharacter_BP to Character_BP to be more generic for creating a project template

Primary Reviewers:
@joshuahuburn @m-samiec

### CODE REVIEW

**Comment by m-samiec** [comment]
Is the `ThirdPersonCharacter.uasset` no longer need at all? I don't see a replacement.

**Comment by jnicholas-io** [comment]
> Is the `ThirdPersonCharacter.uasset` no longer need at all? I don't see a replacement.

It's still there and being used, just modified

File: Game/StarterProject.uproject [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "{594FE353-43A3-F1B1-D054-41BA6971F9AF}",
```
Comment (m-samiec) [inline Game/StarterProject.uproject]: Please revert this change.

File: Game/Config/DefaultEngine.ini [hunk 2]
```diff
@@ -40,6 +40,7 @@ bEnablePCM=False
 bEnableStabilization=False
 bWarnMissingLocks=True
 bEnable2DPhysics=False
+PhysicErrorCorrection=(PingExtrapolation=0.100000,ErrorPerLinearDifference=1.000000,ErrorPerAngularDifference=1.000000,MaxRestoredStateError=1.000000,PositionLerp=0.000000,AngleLerp=0.400000,LinearVelocityCoefficient=100.000000,AngularVelocityCoefficient=10.000000,ErrorAccumulationSeconds=0.500000,ErrorAccumulationDistanceSq=15.000000,ErrorAccumulationSimilarity=100.000000)
```
Comment (joshuahuburn) [inline Game/Config/DefaultEngine.ini]: Just wondering what this is?

File: Game/Config/DefaultEngine.ini [hunk 3]
```diff
@@ -67,5 +70,4 @@ SyncSceneSmoothingFactor=0.000000
 AsyncSceneSmoothingFactor=0.990000
 InitialAverageFrameRate=0.016667
 PhysXTreeRebuildRate=10
-
-
+DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
```
Comment (joshuahuburn) [inline Game/Config/DefaultEngine.ini]: And this :D

File: Game/Config/DefaultEngine.ini (hunk 2 above)
Comment (jnicholas-io) [reply Game/Config/DefaultEngine.ini]: I have no idea :| I'll get this removed
